app/views/cocktails/view.php

```diff
@@ -72,7 +74,7 @@
             <?php endif; ?>
             <!-- Display the like count -->
         </div>
-        <h1 class="title"><?= htmlspecialchars($cocktail->getTitle() ?? 'Untitled') ?></h1>
+        <h1 class="title"><?= ucwords(strtolower($cocktailTitle)) ?></h1>
```

Comment:
no htmlspecialchars applied to output.  XSS vulnerable .

Reply:
Check top of page